x-pack/plugins/monitoring/server/lib/elasticsearch/get_last_recovery.js

```diff
@@ -56,6 +56,7 @@ export function getLastRecovery(req, esIndexPattern) {
   const params = {
     index: esIndexPattern,
     size: 1,
+    rest_total_hits_as_int: true,
```

Comment:
For this one, we can change the handler code to just see if `hits.hits.length === 1` (we only request `size: 1`), then do the work. This would remove the need for this parameter.

Reply:
Sounds good, will adjust